packages/peregrine/lib/store/actions/cart/asyncActions.js

```diff
@@ -336,6 +336,52 @@ export const getCartDetails = payload => {
     };
 };
 
+export const retrieveAndMergeCarts = payload =>
```

Comment:
Please add JSDoc here explaining what this function does.

Reply:
sure